lms/static/scripts/frontend_apps/components/dashboard/CourseActivity.tsx

```diff
@@ -157,7 +168,9 @@ export default function CourseActivity() {
             stats.last_activity && <FormattedDate date={stats.last_activity} />
           );
         }}
-        navigateOnConfirmRow={stats => assignmentURL(stats.id)}
+        navigateOnConfirmRow={stats =>
+          urlWithFilters({ studentIds }, { path: assignmentURL(stats.id) })
+        }
```

Comment:
The duplication between the `<RouterLink>` definition and the `navigateOnConfirmRow` callback is unfortunate, but manageable at present.